ui/app/models/pki/issuer.js

```diff
@@ -29,6 +29,12 @@ export default class PkiIssuerModel extends PkiCertificateBaseModel {
   get useOpenAPI() {
     return false;
   }
+  @attr('string', { displayType: 'masked' }) certificate;
```

Comment:
I'm seeing these attributes on my branch locally - have you pulled the latest changes from main? 

I've noticed that we have repeated attrs in this model and `PkiCertificateBaseModel`. I had tried cleaning up in an earlier PR but it was too challenging with the whole team working on the same project. Maybe we should add a cleanup task to remove duplicate attributes.. 🤔

Reply:
I did pull the latest, but I will try again